specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/LinkedService.json

```diff
@@ -1850,6 +1850,41 @@
         "encryptedCredential": {
           "type": "string",
           "description": "The encrypted credential used for authentication. Credentials are encrypted using the integration runtime credential manager. Type: string."
+        },
+        "allowZeroDateTime": {
+          "type": "object",
+          "x-ms-format": "dfe-bool",
+          "description": "This allows the special “zero” date value 0000-00-00 to be retrieved from the database. Type: boolean."
+        },
+        "connectionTimeout": {
+          "type": "object",
+          "x-ms-format": "dfe-int",
+          "description": "The length of time (in seconds) to wait for a connection to the server before terminating the attempt and generating an error. Type: integer."
+        },
+        "convertZeroDateTime": {
+          "type": "object",
+          "x-ms-format": "dfe-bool",
+          "description": "True to return DateTime.MinValue for date or datetime columns that have disallowed values. Type: boolean."
+        },
+        "guidFormat": {
```

Comment:
>guidFormat [](http://example.com/codeflow?start=9&length=10)

can you come up with a different name for this field. The descriptions says a lot more than that is reflected in 'guidFormat'

Reply:
It is also by design, same with the SDK